### PR TITLE
Update the changelog

### PR DESCRIPTION
**Pull request by glibsm**
It's quite stale

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
-- Add support form `map`, `slice` and `array` to dig
-- Remove `Must*` funcs to greatly reduce API surface area.
-- Remove `*All` funcs to reduce API surface area. Available methods on
-container are `Provide`, `Resolve` and `Invoke`
+
+- **[Breaking]** Remove `Must*` funcs to greatly reduce API surface area.
+- **[Breaking]** Restrict the API surface to only `Provide` and `Invoke`.
 - Providing constructors with common returned types results in an error.
-- Update `Provide` and `Resolve` methods to accept variadic arguments.
+- **[Breaking]** Update `Provide` method to accept variadic arguments.
```
Comment (alsamylkin) [inline CHANGELOG.md]: I think dig is not concurrent safe anymore.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,12 +2,12 @@
 
 
 ## v0.4 (unreleased)
-- Add support form `map`, `slice` and `array` to dig
```
Comment (alsamylkin) [inline CHANGELOG.md]: Why do you delete entries, not append?

File: CHANGELOG.md (hunk 1 above)
Comment (glibsm) [inline CHANGELOG.md]: technically that was never in the changelog in the first place, i.e. there is no reversal of previous feature :P

File: CHANGELOG.md (hunk 2 above)
Comment (glibsm) [inline CHANGELOG.md]: Because we don't *explicitly* support only map and slice anymore, but rather all pointer-like and value types that are no primitives

**Pull request merged**